docs/pages/kubernetes-access/helm/guides/aws.mdx

```diff
@@ -230,7 +248,57 @@ $ kubectl create namespace teleport
 $ kubectl --namespace teleport create -f aws-issuer.yaml
 ```
 
-## Step 5/7. Set values to configure the cluster
+## Step 4b/7. Configure Teleport to use ACM to handle TLS
+
+<Admonition type="warning" title="Command-line Teleport Application access does not work with ACM">
```

Comment:
For me, an Admonition title of this length in all caps is difficult to read. 

Maybe we could move the heading to the first line of the Admonition, then use "Warning" for the Admonition title? (If we do this, we should capitalize "Application Access" since it's a product name.)